### ISSUE TITLE
DependencyInjection inside Job

### ISSUE THREAD

**Issue by smakrem** [icsharp/Hangfire.RecurringJobExtensions#6]
Hello
No way to use depency injection inside a job ? ,because
IRecurringJob  must declare a public   prameterless constructor

Thanks

**Comment by icsharp** [comment]
@smakrem 
yup, the idea/rule is using IRecurringJob to impl the job with DI.
